### PR TITLE
handle situation when _selinux_policy_version is not defined

### PR DESCRIPTION
**Pull request by xsuchy**

### CODE REVIEW

**Review by FrostyX — APPROVED** [review]
The failed CI is because of #64

**Comment by praiskup** [comment]
This is interesting, I see this for the first time.  Do we really want to depend on specific selinux-policy version, and rebuild anytime _they_ bump it?

**Comment by xsuchy** [comment]
The rebuild would be needed only if the version of selinux is decreased. That will never happen.

**Comment by praiskup** [comment]
Ah, ```$ rpm --eval %selinux_requires

Requires: selinux-policy >= 41.16 
BuildRequires: pkgconfig(systemd) 
BuildRequires: selinux-policy 
BuildRequires: selinux-policy-devel 
Requires(post): selinux-policy-base >= 41.16 
Requires(post): libselinux-utils 
Requires(post): policycoreutils 
%if 041 || 0 > 7
Requires(post): policycoreutils-python-utils 
%else 
Requires(post): policycoreutils-python 
%endif 
```

**Comment by xsuchy** [comment]
Updated. I intentionally left it in two commits, so it is visible the interaction and intention.

File: dist-git/dist-git.spec [hunk 1]
```diff
@@ -59,16 +59,10 @@ package sources.
...
-BuildRequires:  selinux-policy-devel
 BuildRequires:  hardlink
 
 Requires:       %name = %version-%release
-%if "%{_selinux_policy_version}" != ""
-Requires:       selinux-policy >= %{_selinux_policy_version}
-%endif
-Requires(post):   /usr/sbin/semodule, /sbin/restorecon
-Requires(postun): /usr/sbin/semodule, /sbin/restorecon
+%{?selinux_requires}
```
Comment (praiskup) [inline dist-git/dist-git.spec:65]: It looks this macro is not yet finished :-( the %selinux_requires is not defined by redhat-rpm-config nor any other package in minimal buildroot. I [reported this problem](https://bugzilla.redhat.com/show_bug.cgi?id=2311871).

The docs are not updated: https://fedoraproject.org/wiki/SELinux/IndependentPolicy#Example_spec_file_changes_to_incorporate_-selinux_subpackage

IOW, we need to do:
BuildRequires: selinux-policy-devel
%{?selinux_requires}

**Comment by xsuchy** [comment]
Updated.

**Review by praiskup — APPROVED** [review]
LGTM

**Pull request merged**